lib/view/account_view.dart

```diff
@@ -25,74 +27,93 @@ Widget accountWithConnectionItem(
+                        child: _blockchainAddressView(
+                          context,
+                          a,
+                          onTap: () => Navigator.of(context).pushNamed(
+                              GlobalReceiveDetailPage.tag,
+                              arguments: a),
+                        ),
+                      ),
+                    ],
+                  ),
+                )
+                .toList(),
+          ],
+        ),
       );
     case 'Connection':
       final connection = categorizedAccounts.accounts.first.connections?.first;
       if (connection == null) return const SizedBox();
-
-      return Row(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Container(
-              alignment: Alignment.topCenter, child: _appLogo(connection)),
-          const SizedBox(width: 24),
-          Expanded(
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                Row(
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: [
-                      Expanded(
-                        child: Text(
-                            connection.name.isNotEmpty
-                                ? connection.name
-                                : "unnamed".tr(),
-                            overflow: TextOverflow.ellipsis,
-                            style: theme.textTheme.headlineMedium),
-                      ),
-                      linkedBox(context),
-                    ]),
-                const SizedBox(height: 8),
-                ...categorizedAccounts.accounts
-                    .map((a) => Container(
-                        child: _blockchainAddressView(context, a,
-                            onTap: () => Navigator.of(context).pushNamed(
-                                GlobalReceiveDetailPage.tag,
-                                arguments: a))))
-                    .toList(),
-              ],
-            ),
+      return ExpandedWidget(
+        header: Padding(
+          padding: ResponsiveLayout.pageEdgeInsets.copyWith(top: 0, bottom: 0),
+          child: Row(
+            children: [
+              SizedBox(
+                width: 24,
+                height: 24,
+                child: _appLogo(connection),
+              ),
+              const SizedBox(width: 32),
+              Expanded(
+                child: Text(
+                    connection.name.isNotEmpty
+                        ? connection.name
+                        : "unnamed".tr(),
+                    overflow: TextOverflow.ellipsis,
+                    style: theme.textTheme.headlineMedium),
+              ),
+              linkedBox(context),
+            ],
           ),
-        ],
+        ),
+        child: Column(
+          children: [
```

Comment:
It can be better with:

``` 
Column(
  children: categorizedAccounts.accounts
                .map((a) => [
                        addOnlyDivider(),
                        Container(
                            child: _blockchainAddressView(context, a,
                                onTap: () => Navigator.of(context).pushNamed(
                                    GlobalReceiveDetailPage.tag,
                                    arguments: a))),
                      ])
                .flattened
                .toList(),
```

Reply:
@ppupha I think we don't need the `[...` since the `.toList()` is a list already.